dj_cqrs/mixins.py

```diff
@@ -404,3 +408,54 @@ def cqrs_delete(cls, master_data):
             raise NotImplementedError
 
         return cls.cqrs.delete_instance(master_data)
+
+    @staticmethod
```

Comment:
This two methods seem to have no association with instance. Do you think it's better to move implementation to manager or to leave here?

Reply:
I think for end library users it's better to stay here because it's easy to:
- implement
- understand
- support backward compatibility